app/src/main/res/layout-land/fragment_edit_keyboard.xml

```diff
@@ -0,0 +1,168 @@
+    android:layout_height="match_parent"
+    tools:background="@color/colorPrimaryDark">
+
+    <com.willowtree.vocable.customviews.VocableImageButton
+        android:id="@+id/back_button"
+        android:layout_width="@dimen/edit_keyboard_button_width"
+        android:layout_height="@dimen/edit_keyboard_button_height"
+        android:layout_marginStart="@dimen/edit_keyboard_horizontal_margin"
+        android:layout_marginTop="@dimen/edit_keyboard_top_margin"
+        android:background="@drawable/button_default_background"
+        android:padding="@dimen/edit_keyboard_padding"
+        android:src="@drawable/category_back_button_icon"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.appcompat.widget.AppCompatEditText
+        android:id="@+id/keyboard_input"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/keyboard_side_margin"
+        android:layout_marginTop="@dimen/edit_keyboard_top_margin"
+        android:background="@null"
+        android:ellipsize="end"
+        android:gravity="start|center_vertical"
+        android:inputType="textCapSentences"
+        android:maxLines="1"
+        android:text="@string/keyboard_select_letters"
+        android:textColor="@color/textColor"
+        android:textSize="@dimen/edit_keyboard_text_size"
+        android:textStyle="bold"
+        app:layout_constrainedWidth="true"
+        app:layout_constraintBottom_toTopOf="@id/keyboard_key_holder"
+        app:layout_constraintEnd_toStartOf="@+id/save_button"
+        app:layout_constraintHorizontal_bias="0"
+        app:layout_constraintHorizontal_chainStyle="packed"
+        app:layout_constraintStart_toEndOf="@id/back_button"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <com.willowtree.vocable.customviews.VocableImageButton
+        android:id="@+id/save_button"
+        android:layout_width="@dimen/edit_keyboard_button_width"
+        android:layout_height="@dimen/edit_keyboard_button_height"
+        android:layout_marginTop="@dimen/edit_keyboard_top_margin"
+        android:layout_marginEnd="@dimen/edit_keyboard_horizontal_margin"
+        android:background="@drawable/button_default_background"
+        android:padding="@dimen/edit_keyboard_padding"
+        android:src="@drawable/ic_save_40dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <GridLayout
+        android:id="@+id/keyboard_key_holder"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginStart="@dimen/keyboard_side_margin"
+        android:layout_marginTop="@dimen/keyboard_keys_top_margin"
+        android:layout_marginEnd="@dimen/keyboard_side_margin"
+        android:layout_marginBottom="@dimen/edit_keyboard_bottom_margin"
+        android:columnCount="10"
+        android:rowCount="3"
```

Comment:
If these values are different for each of the layout types, you should move them into the corresponding `integers.xml` file